inductiva/_cli/cmd_tasks/kill.py

```diff
@@ -0,0 +1,22 @@
+"""Kills a tasks by id via CLI."""
+
+import inductiva
+
+
+def kill_task(args):
+    """Kills a task by id."""
+    inductiva.tasks.Task(args.id).kill(wait_timeout=args.wait_timeout)
+
+
+def register(parser):
+    """Register the kill task command."""
+
+    subparser = parser.add_parser("kill", help="Kill a running task.")
+    subparser.add_argument("id", type=str, help="Id of the task to kill.")
```

Comment:
We should be able to kill multiple tasks at the same time. I suggest setting nargs='+':

```suggestion
    subparser.add_argument("id", type=str, help="ID(s) of the task to kill.", nargs='+')
```

Also, we're missing confirmation request. Something like:

```
Do you confirm you want to kill %d tasks (y/[N])?
```

**IMPORTANT:** check in the CLI document what is the name of the items (`task_id` vs `id`) to make sure we're coherent across all commands.

Reply:
We call it id.